### PR TITLE
Oh god im sorry

### PR DESCRIPTION
**Pull request by Cuboos**
IT'S NOT MY FAULT, It'S NOT MY FAULT. BANDIT MADE ME DO IT.

### CODE REVIEW

File: code/game/machinery/computer/arcade.dm [hunk 1]
```diff
@@ -427,14 +429,17 @@
 			gameStatus = ORION_STATUS_START
 			name = "The Orion Trail"
 			desc = "Learn how our ancestors got to Orion, and have fun in the process!"
+		playsound(src.loc, 'sound/arcade/orion/Ori_fail.ogg', 50, 0, extrarange = -3, falloff = 10)
```
Comment (duncathan) [inline code/game/machinery/computer/arcade.dm]: Orion trail? More like Orion FAIL ohhhhh

**Comment by PKPenguin321** [comment]
you are really bad at describing your PRs
looks like this adds graphics to orion trail?

**Comment by Cuboos** [comment]
ignore it, TortoiseGit fucked up on me.

**Comment by Cheridan** [comment]
DENIED

**Pull request closed without merging**